.changeset/rotten-rivers-wonder.md

```diff
@@ -18,7 +18,7 @@ const machine = createMachine(
   },
   {
 -   activities: {
-+   behaviors: {
++   actors: {
```

Comment:
a good change 👍 you still mention behaviors in a lot of places, I wonder if we want to stick with that terminology? I see it as potentially very confusing for people, at the very least maybe we should bury it deep down and only mention it in like "the very advanced" section of the docs? 😅